corehq/warehouse/transforms/__init__.py

```diff
@@ -0,0 +1,17 @@
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+)
+
+
+def app_status_transform(row):
+    print row
```

Comment:
this will actually fetch things like the commcare version that are hard to get in SQL